Kronos-Shifts-Connector/Microsoft.Teams.Shifts.Integration/Microsoft.Teams.Shifts.Integration.API/Controllers/OpenShiftController.cs

```diff
@@ -103,9 +110,9 @@ public async Task<ShiftsIntegResponse> CreateOpenShiftFromTeamsAsync(Models.Inte
 
             var openShiftDetails = new
             {
-                KronosStartDateTime = this.utility.UTCToKronosTimeZone((DateTime)(openShift.DraftOpenShift?.StartDateTime ?? openShift.SharedOpenShift?.StartDateTime), team.KronosTimeZone),
-                KronosEndDateTime = this.utility.UTCToKronosTimeZone((DateTime)(openShift.DraftOpenShift?.EndDateTime ?? openShift.SharedOpenShift?.EndDateTime), team.KronosTimeZone),
-                DisplayName = openShift.DraftOpenShift?.DisplayName ?? openShift.SharedOpenShift?.DisplayName ?? null,
+                KronosStartDateTime = this.utility.UTCToKronosTimeZone((DateTime)openShift.SharedOpenShift?.StartDateTime, team.KronosTimeZone),
+                KronosEndDateTime = this.utility.UTCToKronosTimeZone((DateTime)openShift.SharedOpenShift?.EndDateTime, team.KronosTimeZone),
+                DisplayName = openShift.SharedOpenShift?.DisplayName ?? null,

```

Comment:
Just assign DisplayName = openShift.SharedOpenShift.DisplayName because if it is null then you will be assigning null and don't need the null coalescing operator. (you will have already validated that SharedOpenShift is not null on entry as per the previous comment)